pkg/restapi/vc/operation/operations.go

```diff
@@ -97,12 +112,33 @@ func New(provider storage.Provider, client Client, kms legacykms.KMS, vdri vdria
 		return nil, fmt.Errorf("failed to instantiate new csl status: %w", err)
 	}
 
+	cryptoProv := cryptoProvider{
+		crypto: kms,
```

Comment:
it is more clear when used here... a kms is not a crypto instance

Reply:
I see what you mean! I've updated the code.